cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -424,7 +424,7 @@
 
     <property>
         <name>data.tx.snapshot.interval</name>
-        <value>300</value>
+        <value>60</value>
```

Comment:
It actually should belongs to previous PR that changes state store to use dataset. A shorter interval for taking tx snapshot is more favorable since the queue eviction coprocessor now needs to read the state store transactionally, meaning a more up-to-date tx snapshot helps.